tuist/tuist
# File generation for Entitlements

Discussion here: https://github.com/tuist/tuist/discussions/5349

### Short description 📝

Extracting and extending the current `InfoPlist` generation logic for `Entitlements`, allowing users to declare these inside of Tuists project files. 

### How to test the changes locally 🧐

Add entitlements to your target and run `tuist generate`: 

```swift
var entitlements: [String: Plist.Value] = [
    "aps-environment": "development",
    "com.apple.developer.default-data-protection": "NSFileProtectionCompleteUntilFirstUserAuthentication",
]

Target(
    // ...
    entitlements: entitlements
    // ...
)
```

But also just use the current method of importing a file, which should also work. 

### Contributor checklist ✅


### Reviewer checklist ✅
